xml/System.Windows.Forms.VisualStyles/FontProperty.xml

```diff
@@ -46,6 +46,7 @@
       <MemberValue>2601</MemberValue>
       <Docs>
         <summary>The font that a glyph is drawn with.</summary>
+        <remarks>To be added.</remarks>
```

Comment:
So this is a tiny bug in the tool, in which if it does not find a `<remarks>` item in the dotnet-api-docs xml, it will add it with the default boilerplate text.

It's harmless, but if you prefer to remove this (and all other instances) so the PR is smaller, feel free to do it. Totally optional.

I will get it fixed in the tool as soon as I get a chance.

Reply:
I see you fixed this in the tool. I'll regenerate these files. :)

Reply:
@carlossanlop I used the latest version of the tool, but it seemed to add lots of remarks now regarding explicit interface implementations. Is this intended?

Reply:
It wasn't intended but it doesn't hurt.
I just fixed that bug in the newest version. It should not happen again.